samples/drivers/display/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _display-sample:
+
+Display Sample
+##############
+
+Overview
+********
+
+This sample will draw some basic rectangles onto the display.
+The rectangle colors and positions are chosen so that you can check the
+orientation of the LCD and correct RGB bit order. The rectangles are drawn
+in clockwise order, from top left corner: red, green, blue, grey. The shade of
+grey changes from black through to white. If the grey looks too green or red
+at any point or the order of the corners is not as described above then the LCD
+may be endian swapped.
+
+Building and Running
+********************
+
+As this is a generic sample it should work with any display supported by Zephyr.
+
+Below is an example on how to build for a :ref:`nrf52840_pca10056` board with a
+:ref:`adafruit_2_8_tft_touch_v2`.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/display_shield
+   :board: nrf52840_pca10056
+   :goals: build
+   :shield: adafruit_2_8_tft_touch_v2
+   :compact:
+
+For testing purpose without the need of any hardware, the :ref:`native_posix`
+board is also supported and can be built as follows;
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/display_shield
+   :board: native_posix
+   :goals: build
+   :compact:
+
+List of Arduino-based display shields
+*************************************
+
+- :ref:`adafruit_2_8_tft_touch_v2`
+- :ref:`ssd1306_128x32_shield`
```

Comment:
This has to be fixed to pass Documentation checks:

```diff
diff --git a/samples/drivers/display/README.rst b/samples/drivers/display/README.rst
index 99a66aa2a3..731de70ca2 100644
--- a/samples/drivers/display/README.rst
+++ b/samples/drivers/display/README.rst
@@ -42,6 +42,5 @@ List of Arduino-based display shields
 *************************************
 
 - :ref:`adafruit_2_8_tft_touch_v2`
-- :ref:`ssd1306_128x32_shield`
-- :ref:`ssd1306_128x64_shield`
+- :ref:`ssd1306_128_shield`
 - :ref:`waveshare_e_paper_raw_panel_shield`
```

Reply:
thx for the input, doc has been updated